src/tables/table.py

```diff
@@ -214,6 +214,8 @@ def _write_table(self, df, conflict_columns=None):
 
         sql = "".join(["INSERT INTO ", self._schema, ".", self._table_name,
                        " (", columns, ") VALUES ", values])
+
+        print("Writing to: ", self._schema, '.', self._table_name)
         if conflict_columns:
```

Comment:
Could this please use IOs._print() so it can be consistent and we can easily wrap logger into IOs.

Reply:
I think this can be made a separate sub-issue? There is lots of places this needs to be changed

Reply:
Sure thing, I'll go add it to the gluing things together issue.